client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -47,23 +57,31 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
   const [jurisdictionFile] = useJurisdictionFile(electionId)
   const [contests] = useContests(electionId)
   const [sampleSizeOptions, setSampleSizeOptions] = useState<
-    IStringSampleSize[]
-  >([])
-  const [sampleSize, setSampleSize] = useState('')
+    IStringSampleSizes
+  >({})
+  const [sampleSizes, setSampleSizes] = useState<IFormOptions>({})
   const history = useHistory()
   const [isConfirmDialogOpen, setIsConfirmDialogOpen] = useState(false)
 
   useEffect(() => {
     ;(async () => {
       try {
-        const { sampleSizes }: { sampleSizes: ISampleSizeOption[] } = await api(
+        const {
+          sampleSizes: options,
+        }: { sampleSizes: ISampleSizeOptions } = await api(
```

Comment:
what do you think about pulling this out into a hook and defining the type there? like the pattern we've been using for other API data sets

Reply:
Probably a good idea. It would clean things up significantly and keep in pattern with everything else. Not sure why this one wasn't pulled out already, actually.